.vscode/launch.json

```diff
@@ -1,21 +1,16 @@
 {
-    // Use IntelliSense to learn about possible attributes.
-    // Hover to view descriptions of existing attributes.
-    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    // https://code.visualstudio.com/docs/editor/debugging#_launch-configurations
     "version": "0.2.0",
     "configurations": [
         {
             "name": "gcc build then debug file",
             "type": "cppdbg",
             "request": "launch",
-            "program": "${fileDirname}/a.out",
-            "args": [
-                "test-1.txt",
-                "test-2.txt",
-                "test-3.txt"
-            ],
+            "program": "${workspaceFolder}/a.out",
+            // Pass arguments to the executable: [ "foo", "bar" ] will run './a.out foo bar'.
+            "args": [],
             "stopAtEntry": false,
-            "cwd": "${fileDirname}",
+            "cwd": "${workspaceFolder}",
```

Comment:
This should prevent the debugger from being picky about which file you can press F5 from to build+debug. Before, it was relying on the directory name of the currently selected file.

If you had a VS Code config file selected, it would try to start debugging in the `.vscode` directory and fail.